aten/src/ATen/native/README.md

```diff
@@ -1,7 +1,16 @@
 ATen native functions are a mechanism to write ATen methods which only
 make use of other ATen operations (e.g., it is not necessary to bind into
 TH/THC code).  These functions are declared in this file and then folded
-into the ATen code generation process.
+into the ATen code generation process, so that they become available
+either as methods on `Tensor` (`t.mymeth()`) and functions in the ATen
+namespace (`at::myfunc()`).
+
+In PyTorch, every native function gets Python bindings generated,
+making them available as methods on `torch._C._VariableBase`
```

Comment:
I think there is too much implementation detail here, it's guaranteed to get out of date.  I'd just say that methods end up on Variables in pytorch.  For functions I don't know what "subsequently" rebound to the Python module means here -- do you mean the user is reasonable for doing that if they want?

Reply:
I think I commented on the old commit here, but there is similar wording in the latest commit.